cleanup_env.py

```diff
@@ -0,0 +1,228 @@
+""" Utility to cleanup instance/node along with its volumes"""
```

Comment:
hope this `--cleanup` option helpful?  https://github.com/red-hat-storage/cephci/blob/master/ceph/utils.py#L190 

I'm not sure it works with error/failed state. But we could make use of this code instead of creating new module.

Reply:
Sure sunil